components/Banner/examples/supplemental/MDCBannerViewConstants.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
This will be moved to `/private` right?

Reply:
Right. I created an issue and added some comment to make sure it will happen later. Addressed in https://github.com/material-components/material-components-ios/pull/6830/commits/3a9c5afbbf966c8a2ef5a6e8ad5d76641534b057#diff-860ee71b623e164ecfe00e2b3f510eaaR15.